packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -716,15 +716,6 @@ class DirectStripeRoutes {
     }
 
     const { paymentMethodId } = request.payload;
-
-    // Verify the payment method is already attached to the customer
-    const alreadyAttached = !!customer.sources.data.find(
```

Comment:
Had to disable this check - the new Payment Methods aren't listed in customer sources, and the updateDefaultPaymentMethod call will throw if it's an invalid Payment Method anyway